packages/repl/src/lib/Output/PaneWithPanel.svelte

```diff
@@ -121,7 +121,8 @@
 	}
 
 	.panel-body {
-		overflow: auto;
+		overflow: scroll;
+		max-height: 100%;
```

Comment:
This fixes it but makes it bigger than its current size because it fills the height of the container and doesn't take into account the height of the console controls we have at the top. This causes some of the console logs to be hidden. I wonder if we can easily fix this by wrapping the panel body in another container.